go-controller/pkg/libovsdbops/template_var.go

```diff
@@ -0,0 +1,121 @@
+package libovsdbops
+
+import (
+	"context"
+
+	libovsdbclient "github.com/ovn-org/libovsdb/client"
+	libovsdb "github.com/ovn-org/libovsdb/ovsdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+)
+
+type chassisTemplateVarPredicate func(*nbdb.ChassisTemplateVar) bool
```

Comment:
I am just thinking, can we consider ChassisTemplateVar an extension of chassis, move all of this to `chassis.go` and don't have `DeleteChassisTemplateVar` and basically do that in `DeleteChassis` and `DeleteChassisWithPredicate` as we already do with Chassis/ChassisPrivate?

Reply:
I actually think it is a good idea to delete ChassisTemplateVar in the same transaction as Chassis if they are supposed to always happen together.